test/Frontend/crash-in-user-code.swift

```diff
@@ -12,6 +12,7 @@
 // CHECK: Stack dump:
 // CHECK-NEXT: Program arguments:
 // CHECK-NEXT: Swift version
+// CHECK-NEXT: Compiling with Swift version
```

Comment:
This is part of the existing output right?

Reply:
No, which also answers your question below - the driver current outputs "Swift version ..." but does so before parsing any flags. I added a second version output after flags are passed so that we can pass in the effective version as well, which is the "Compiling with..." line (though note some tests are failing because apparently this can actually be Apple Swift sometimes).

I could remove the first version, but crashes during argument parsing would be missing the version then. Thoughts?

Reply:
It looks we only want the effective language version out of that line and from what I see `swift::getSwiftFullVersion` directly prints the value of `LangOpts.EffectiveLanguageVersion`. How about we bring that logic up and print the language version on its own line only if it's different than `getCurrentLanguageVersion`?